index.html

```diff
@@ -48,30 +51,182 @@ <h2>I'm a Software Developer</h2>
+              <a href="https://github.com/fpsapc" target="_blank" rel="noopener"> <img class="social1" src="/img/Disabled.png" alt="" /></a>
             </li>
             <li>
-              <a href="https://linkedin.com/in/salman-ahmad1987" target="_blank"> <img class="social" src="/img/Disabled-1.png" alt="" /></a>
+              <a href="https://linkedin.com/in/salman-ahmad1987" target="_blank" rel="noopener"> <img class="social1" src="/img/Disabled-1.png" alt="" /></a>
             </li>
             <li>
-              <a href="https://www.linkedin.com/company/vivimedia/about/" target="_blank"> <img class="social" src="/img/Enabled.png" alt="" /></a>
+              <a href="https://www.linkedin.com/company/vivimedia/about/" target="_blank" rel="noopener"> <img class="social1" src="/img/Enabled.png" alt="" /></a>
             </li>
             <li>
-              <a href="https://twitter.com/salman-ahmadkhan1987" target="_blank"> <img class="social" src="/img/Disabled-2.png" alt="" /></a>
+              <a href="https://twitter.com/salman-ahmadkhan1987" target="_blank" rel="noopener"> <img class="social1" src="/img/Disabled-2.png" alt="" /></a>
             </li>
             <li>
-              <a href="mailto:fpsapc@gmail.com" target="_blank"> <img class="social" src="/img/Disabled-3.png" alt="" /></a>
+              <a href="mailto:fpsapc@gmail.com" target="_blank" rel="noopener"> <img class="social1" src="/img/Disabled-3.png" alt="" /></a>
             </li>
-            ...
           </ul>
           </div>
         <img class="backgroundimg" src="/img/Illustration.png" alt="" />
       </section>
+      <section class="work">
+       <h1 id="work-title">My Recent Works</h1>
+       <img id="work-image" src="/img/odinproject.jpg" alt="">
+       <h2 class="work-subtitle">Multi-Post Stories</h2>
+       <p class="work-details">A daily selection of privately personalized reads; o accounts or sign-ups required. has been the idustry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+       <div class="flex">
+        <p class="flex-item item1">css</p>
+        <p class="flex-item item2">html</p>
+        <p class="flex-item item3">bootstrap</p>
+        <p class="flex-item item4">Ruby</p>
+       </div>
+       <button class="work-button">See Project</button>
+      </section>
+      <section class="projects">
+        <div id="project1">
+          <h2 class="work-subtitle protitle">Te Odin Project</h2>
+          <p class="work-details prodetails">My first project when I start my web development journey</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
```

Comment:
- [ ] The cards are not equal in length and it is because you are styling them differently, I would implore you to use the same class name for them and then style them together.